libraries/abstractions/common_io/include/iot_uart.h

```diff
@@ -159,7 +160,7 @@ typedef struct
  *  IotUARTHandle_t xOpen;
  *  int32_t lRead, lWrite, lClose;
  *  BaseType_t xCallbackReturn;
- *  uint8_t ucPort = 1; /* Each UART peripheral will be assigned an integer.
+ *  uint8_t ucPort = 1; // Each UART peripheral will be assigned an integer.
```

Comment:
Why switch to C++ comment style here? I'm fine with deferring to Uncrustify, as long as we're consistent.

Reply:
Uncrustify does not check examples inside of comments AFAIK. I think the problem is that C -comment style with /* cannot be nested, which means that if yo uwant to put an example in your header comment block you have to do it this way or it does not work. I am not aware of a better way to do this today.

Reply:
Yeah, this is just comment of example code. Nested "/* */" doesn't work